<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainwindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="forgot_password">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Click here to reset your password.</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#forgot_password {
	font-size: 15px;
	background-color: transparent;
	color: black;
}

#forgot_password:hover {
	color: #FFF;
}</string>
    </property>
    <property name="text">
     <string>Forgot password?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="help_button">
    <property name="geometry">
     <rect>
      <x>534</x>
      <y>30</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Help</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="passwordGroup">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>251</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="loginPasswordInput">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lato</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Enter your valid password.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">#loginPasswordInput {
	padding: 10px;
	border-top-right-radius: 10px;
	border-bottom-right-radius: 10px;
	font-size: 20px;
	color: #374957;
}

#loginPasswordInput:focus {
	background-color:#E3E8F0;
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="loginPasswordIcon">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-top-left-radius: 10px;
border-bottom-left-radius: 10px;
font-size: 20px;
qproperty-alignment: AlignCenter;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="buddy">
      <cstring>loginPasswordInput</cstring>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="usernameGroup">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>181</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="loginUsernameIcon">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-top-left-radius: 10px;
border-bottom-left-radius: 10px;
font-size: 20px;
qproperty-alignment: AlignCenter;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="buddy">
      <cstring>loginUsernameInput</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="loginUsernameInput">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>0</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lato</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Enter your valid username.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">#loginUsernameInput {
	padding: 10px;
	background-color: #fff;
	border-top-right-radius: 10px;
	border-bottom-right-radius: 10px;
	font-size: 20px;
	color: #374957;
}

#loginUsernameInput:focus {
	background-color: #E3E8F0;
}</string>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="loginHeader">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>93</y>
      <width>221</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: gold;
font-size: 35px;</string>
    </property>
    <property name="text">
     <string>USER LOGIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>301</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Click here to login.</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#loginButton {
	background-color: #32D736;
	border-radius: 10px;
	color: #fff;
	font-size: 20px;
	padding: 8px;
}

#loginButton:hover {
	background-color: #34E038;
}


</string>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="noAccountGroup">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>138</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="createAccount">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lato</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Click here to create a new account.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">#createAccount {
	font-size: 15px;
	background-color: transparent;
	color: #32D736;
}

#createAccount:hover {
	color: #3EF642;
}</string>
     </property>
     <property name="text">
      <string>Create one</string>
     </property>
    </widget>
    <widget class="QLabel" name="noAccountMessage">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>0</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lato</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 15px;
background-color: transparent;
color: red;</string>
     </property>
     <property name="text">
      <string>Don't have an account?</string>
     </property>
     <property name="buddy">
      <cstring>createAccount</cstring>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="icon">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>3</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="KULOGO">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>3</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>forgot_password</zorder>
   <zorder>help_button</zorder>
   <zorder>passwordGroup</zorder>
   <zorder>usernameGroup</zorder>
   <zorder>loginButton</zorder>
   <zorder>noAccountGroup</zorder>
   <zorder>loginHeader</zorder>
   <zorder>icon</zorder>
   <zorder>KULOGO</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
